src/components/header/Header.css

```
@@ -0,0 +1,71 @@
+.yes-bank-header {
+  background-color: rgb(0, 0, 0);
+  background: linear-gradient(#1d1d1d, #000000);
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)